src/assessments/seo/KeyphraseDistributionAssessment.js

```diff
@@ -158,20 +156,18 @@ class KeyphraseDistributionAssessment extends Assessment {
 	 * @returns {Array} All markers for the current text.
 	 */
 	getMarks() {
-		return this._keyphraseDistribution.sentencesToHighlight.map( function( sentence ) {
-			return new Mark( { original: sentence, marked: addMark( sentence ) } );
-		} );
+		return this._keyphraseDistribution.sentencesToHighlight;
 	}
 
 	/**
-	 * Checks whether the paper has a text with at least 200 words and a keyword.
+	 * Checks whether the paper has a text with at least 10 words and a keyword.
 	 *
 	 * @param {Paper} paper The paper to use for the assessment.
 	 *
-	 * @returns {boolean} True when there is a keyword and a text with 200 words or more.
+	 * @returns {boolean} True when there is a keyword and a text with 10 words or more.
 	 */
 	isApplicable( paper ) {
-		return paper.hasText() && paper.hasKeyword() && countWords( paper.getText() ) >= 200;
+		return paper.hasText() && paper.hasKeyword() && getSentences( paper.getText() ).length >= 10;
```

Comment:
Unless all of your sentences are very long, this will make the assessment also trigger on short texts. Currently we trigger when there are 200 + words. Maybe increase to 15 sentences?